bootstrap-salt.sh

```diff
@@ -6342,7 +6350,7 @@ install_opensuse_stable() {
 }
 
 install_opensuse_git() {
-
+    
```

Comment:
This is a minor catch. I don't think we need any blank lines here, with trailing blanks especially.